### PR TITLE
Add support for valve stepper

### PR DESCRIPTION
**Pull request by aksell**
Add valve stepper control  and do some pep8 formatting of python scripts. I am not sure using the `d-pad` on the xbox controller is the best solution to control the valve stepper. Any thoughts on this?

### CODE REVIEW

**Comment by aksell** [comment]
The vertical `d-pad` might be nice to use for tilting the front camera. Should probably not use it for the valve stepper.

**Comment by mortenfyhn** [comment]
Regarding which button to use: Just let people try to sail the rov using different button options and pick the one they like, and merge this without worrying about the button choice now.

File: manipulator_interface/scripts/manipulator_interface.py [hunk 1]
```diff
@@ -76,9 +94,18 @@ def callback(self, msg):
...
+                self.valve_stepper.enable()
+
+
 if __name__ == '__main__':
     try:
         manipulator_interface = ManipulatorInterface()
         rospy.spin()
     except rospy.ROSInterruptException:
         pass
+nn
```
Comment (mortenfyhn) [inline manipulator_interface/scripts/manipulator_interface.py]: ?

**Review by mortenfyhn — COMMENTED** [review]
Looks good, but what's the `nn` thing? Also, has it been tested on hardware?

File: manipulator_interface/scripts/manipulator_interface.py (hunk 1 above)
Comment (aksell) [reply manipulator_interface/scripts/manipulator_interface.py]: I don't know what the nn is, i will remove it. It has been tested on hardware. Thanks.

**Comment by aksell** [comment]
The nn must have been a typo, I removed it. It has been tested on hardware. Thanks.

**Review by mortenfyhn — APPROVED** [review]

**Review by sindrehan — COMMENTED** [review]
Code seems fine, but why is it called valve stepper? We're not turning any valves this year, are we?

**Comment by aksell** [comment]
Huh, i dont know. I guess the handles we are going to turn with the downwards facing stepper just feelt like a valve. Would vertical_stepper maybe be better?

**Review by sindrehan — APPROVED** [review]

**Pull request merged**